src/javascript/binary/static/profittable.js

```diff
@@ -0,0 +1,14 @@
+
+pjax_config_page("profit_tablews", function(){
+    return {
+        onLoad: function() {
+            Content.populate();
```

Comment:
you don't need to populate it as your ids are in your js only. Populate update the static content on trading page